model-optimizer/extensions/ops/bucketize.py

```diff
@@ -17,39 +17,54 @@
 import numpy as np
 
 from mo.graph.graph import Node, Graph
+from mo.middle.passes.convert_data_type import np_data_type_to_destination_type
 from mo.ops.op import Op
 
 
 class Bucketize(Op):
     op = 'Bucketize'
 
     def __init__(self, graph: Graph, attrs: dict):
+        assert 'output_type' in attrs, 'Bucketize has mandatory `output_type` attribute'
```

Comment:
Have you tried to pass the MO IR reader the model where the Bucketize was with `extension` version? I believe that the MO IR reader will fail to save such a model because there is no `output_type` attribute.

Reply:
Recently (~2 weeks ago) ir reader check passed

Reply:
Done

Reply:
Fixed. Now both IRs (`extension` and `opset3`) pass ir reader check.